src/reference/architecture/ui-server.rst

```diff
@@ -53,6 +53,27 @@ Users can only monitor and control their own workflows.
    :align: center
 
 
+Jupyter Hub
+-----------
+
+.. _Cylc Hub configuration file: https://github.com/cylc/cylc-uiserver/blob/master/cylc/uiserver/jupyter_config.py
+
+`Jupyter Hub`_ is a multi-user server which spawns and manages a configured
+service for authenticated users.
+
+The "Cylc Hub" is a Jupyter Hub instance which is pre-configured to spawn
+Cylc UI Servers, launched by the ``cylc hub`` command.
+It is also possible to configure Jupyter Hub yourself, see the Cylc Hub
+configuration file for more information.
```

Comment:
It seems to me that this overlaps/repeats what is written under Multi-User Mode below?

Reply:
This bit describes Jupyter Hub the next bit describes how Cylc multi-user-mode works.

Tis the best I could come up with quickly.